src/main/java/mate/academy/intro/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.intro.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
consider using unchecked exceptions
```suggestion
public class RegistrationException extends RuntimeException {
```